client/main.lua

```diff
@@ -1,9 +1,243 @@
 local config = require 'config.client'
+local sharedConfig = require 'config.shared'
+isLoggedIn = LocalPlayer.state.isLoggedIn
+isDrilling = false
 
 function DropFingerprint()
     if IsWearingGloves() then return end
     if config.fingerprintChance > math.random(0, 100) then
         local coords = GetEntityCoords(cache.ped)
         TriggerServerEvent('evidence:server:CreateFingerDrop', coords)
     end
-end
\ No newline at end of file
+end
+
+AddEventHandler('onResourceStop', function(resource)
+    if resource ~= cache.resource then return end
+    ResetBankDoors()
+end)
+
+RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
+    local config1, config2, config3 = lib.callback.await('qb-bankrobbery:server:GetConfig', false)
+    sharedConfig.powerStations = config1
+    sharedConfig.bigBanks = config2
+    sharedConfig.smallBanks = config3
+    ResetBankDoors()
+    isLoggedIn = true
+end)
+
+RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
+    isLoggedIn = false
+end)
+
+RegisterNetEvent('qb-bankrobbery:client:setBankState', function(bankId)
+    if bankId == 'paleto' then
+        sharedConfig.bigBanks.paleto.isOpened = true
+        OpenPaletoDoor()
+    elseif bankId == 'pacific' then
+        sharedConfig.bigBanks.pacific.isOpened = true
+        OpenPacificDoor()
+    else
+        sharedConfig.smallBanks[bankId].isOpened = true
+        OpenFleecaDoor(bankId)
+    end
+end)
+
+RegisterNetEvent('qb-bankrobbery:client:setLockerState', function(bankId, lockerId, state, bool)
+    if bankId == 'paleto' then
+        sharedConfig.bigBanks.paleto.lockers[lockerId][state] = bool
+    elseif bankId == 'pacific' then
+        sharedConfig.bigBanks.pacific.lockers[lockerId][state] = bool
+    else
+        sharedConfig.smallBanks[bankId].lockers[lockerId][state] = bool
+    end
+end)
+
+--- This will trigger the sequence of opening a safe locker of a bank
+--- @param bankId string | number
+--- @param lockerId number
+--- @return nil
+function openLocker(bankId, lockerId) -- Globally Used
```

Comment:
Is this supposed to be a local function? Either rename to OpenLocker or make a local function